vars/waitForContainer.groovy

```diff
@@ -0,0 +1,19 @@
+def call(String CONTAINER_NAME, String LOGS) {
+    sh """
+        set -o errexit
+        set -o xtrace
+        export CONTAINER_NAME=${CONTAINER_NAME}
+        export LOGS="${LOGS}"
+        attempt=0
+        while [ \$attempt -le 30 ]; do
+            attempt=\$(( \$attempt + 1 ))
```

Comment:
With change to `sleep 10` this loop will be executed for 300 seconds.So I suggest setting `\$attempt -le 4`

Reply:
yes, missed this, addressed, please check